components/formats-bsd/src/loci/formats/AxisGuesser.java

```diff
@@ -71,18 +73,24 @@
   public static final int S_AXIS = 4;
 
   /** Prefixes indicating space dimension. */
-  public static final String[] Z_PREFIXES = {
+  public static final ImmutableSet<String> Z_PREFIXES = ImmutableSet.of(
```

Comment:
Fair enough. I had looked for other usages of guava across the code. Despite being consumed in several places, there is no precedent of using google objects as public API.